usl/usl_kevm.h

```diff
@@ -0,0 +1,241 @@
+#ifndef USL_KEVM_H
+#define USL_KEVM_H
+
+#include <cstdint>
+#include <ctime>
+#include <optional>
+#include <string>
+#include <vector>
+
+namespace usl_kevm {
+
+// Common data types
+using bytes_t = std::string;
+using nonce_t = std::uint64_t;
+using ether_t = std::uint64_t;
+using gas_t = std::uint64_t;
+using hash_t = std::uint64_t;
```

Comment:
I believe most of these hashes are keccak hashes so they are also either u256 or a 32 byte array

Reply:
We should probably double check the exact length of each of the hashes though because I do not believe they are all the same.

Reply:
I checked and all the places where `hash_t` is needed are indeed keccak-256 hashes. Added appropriate comment.